instrumentation/hibernate/hibernate-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/hibernate/HibernateSingletons.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.hibernate;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+
+public class HibernateSingletons {
+
+  private static final Instrumenter<String, Void> INSTANCE;
+
+  static {
+    INSTANCE =
+        Instrumenter.<String, Void>newBuilder(
+                GlobalOpenTelemetry.get(), "io.opentelemetry.hibernate-common", s -> s)
```

Comment:
Nit: WDYT about converting this to `HibernateInstrumenterFactory` with a factory method accepting the instrumentation name? Every hibernate module would pass its own name with the version